packages/lumx-react/src/components/slideshow/Demos.stories.tsx

```diff
@@ -3,3 +3,4 @@ export default { title: 'LumX components/slideshow/Demos' };
 export { default as Auto } from './demos/auto';
 export { default as Default } from './demos/default';
 export { default as Group } from './demos/group';
+export { default as ManualControls } from './demos/externalControls';
```

Comment:
Demos must be referenced in a MDX documentation. Other stories should be added in another file (like `Slideshow.stories.tsx`)
We import demos into storybook to make simplify the development in LumX react but demos belong in the demo site.